owl/SOMA-ACT.owl

```diff
@@ -528,6 +587,26 @@ Induction is often described as a move from many &quot;specifics&quot; (swan A i
     
 
 
+    <!-- http://www.ease-crc.org/ont/SOMA.owl#MakingInteriorAccessible -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/SOMA.owl#MakingInteriorAccessible">
+        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/SOMA.owl#ModifyingPhysicalObject"/>
+        <owl:disjointWith rdf:resource="http://www.ease-crc.org/ont/SOMA.owl#MakingInteriorInaccessible"/>
+        <rdfs:comment>A task in which an Agent will affect a container such that the container's interior is accessible for furhter tasks. The MakingInteriorAccessible task will include locating the container, navigating to it, and manipulating it appropriately to enact this state change.</rdfs:comment>
```

Comment:
How is this different from an Opening task? I guess the idea is to just classify the manipulation itself as opening?

My concern is that this seems very specific and would rather belong in the Plan section where we can represent concrete workflows of steps following each other as you have done in the comment. in my intuition the workflow described in the comment is just one of many ways of executing an opening task. One could say that the activity that executes this task has several motion phases, and in the case of the workflow above these would be navigation, approaching, grasping, etc. but we would only use one task label for all the events (however different motion labels)

Reply:
The rationale behind this being a task is the following:

there exists a CRAM task called Accessing (find it in Seba's list). This is a complex task including navigating to a container and includes Opening as a subtask. To the CRAM task, we now have the MakeInteriorAccessible task as a correspondent.

Naturally, this suggested the need for its opposite, and hence MakeInteriorInaccessible was born. I wouldn't make a distinction between "Plan" and "Task" since these are, as we discussed before, not very well-defined distinctions. You can always split a task into several subtasks; conversely, maybe the level of granularity at which you describe an action is in terms of goals such as making container interiors (in)accessible, without caring for the specifics of manipulation of how to render them thus.

Reply:
I mean, I wouldn't think it is a good idea to have certain CRAM "tasks" correspond to "Tasks" and others to "Plans" just on the basis of some increased complexity in terms of "subtasking" relations. Of course, Plans and Tasks are different ;)

Reply:
> there exists a CRAM task called Accessing (find it in Seba's list). This is a complex task including navigating to a container and includes Opening as a subtask. To the CRAM task, we now have the MakeInteriorAccessible task as a correspondent.

But the act of "accessing" is not the same as "making accessible". In my opinion only the "opening" subtask corresponds to "making accessible"

Reply:
> The rationale behind this being a task is the following:
> 
> there exists a CRAM task called Accessing (find it in Seba's list). This is a complex task including navigating to a container and includes Opening as a subtask. To the CRAM task, we now have the MakeInteriorAccessible task as a correspondent.
> 
> Naturally, this suggested the need for its opposite, and hence MakeInteriorInaccessible was born. I wouldn't make a distinction between "Plan" and "Task" since these are, as we discussed before, not very well-defined distinctions. You can always split a task into several subtasks; conversely, maybe the level of granularity at which you describe an action is in terms of goals such as making container interiors (in)accessible, without caring for the specifics of manipulation of how to render them thus.

You haven't really answered my question how this is different from opening.
So my suggestion is still to stick to Opening and Closing, and these can be subtasks of accessing according to some plan where e.g. something is accessed from a container that is opened first, and closed again afterwards

Reply:
> I mean, I wouldn't think it is a good idea to have certain CRAM "tasks" correspond to "Tasks" and others to "Plans" just on the basis of some increased complexity in terms of "subtasking" relations. Of course, Plans and Tasks are different ;)

Well I was mostly referring to the comment wrt "plan" because I think a task should not be defined by the workflow that executes them but by the roles objects play and by the goal. however it should leave open how the goal is achieved, this is defined in plans

Reply:
The CRAM task Accessing (to which MakeInteriorAccessible is intended to correspond) does however include both getting to the container and the Opening manipulation task. That's the difference.

If you think this partitioning is somehow too CRAM-specific, I'm happy to move MakeInterior(In)Accessible to cram_knowledge, but I don't think it really is as CRAM-specific as it seems. The problem rather comes from the colloquial use of "open"; it can refer to either the full action of going to the container and opening it, or to just the manipulation to open it once there. CRAM makes a distinction between the two, and it's a distinction worth reflecting in SOMA.